SpiffWorkflow/bpmn/PythonScriptEngine.py

```diff
@@ -97,10 +38,14 @@ class PythonScriptEngine(object):
     expressions in a different way.
     """
 
-    def __init__(self, default_globals=None, scripting_additions=None):
-
-        self.globals = default_globals or {}
-        self.globals.update(scripting_additions or {})
+    def __init__(self, default_globals=None, scripting_additions=None, environment=None):
```

Comment:
If the environment is passed in, why would continue to accept default_globals and default_additions as separate arguments?

Reply:
for backwards compatibility of course.

Reply:
Maybe this should log some sort of deprecation warning, so we can move people away from this in the future?

Reply:
Good call, will do.

Reply:
Added the warning and updated tests that used default_globals/scripting_additions.